Test/Publish-WhiskeyNodeModule.Tests.ps1

```diff
@@ -264,6 +276,13 @@ Describe 'PublishNodeModule.when email address property missing' {
     ThenTaskFailed '\bEmailAddress\b.*\bmandatory\b'
 }
 
+Describe 'PublishNodeModule.when publishing node module with initialization mode' {

```

Comment:
This should be failing. Add an assertion that Node.js was actually installed.

Reply:
Done, I ended up Asserting the Mock "Install-whiskeyNodejs" was called